readme.md

```diff
@@ -1,34 +1,35 @@
-![Stripe](https://stripe.com/img/navigation/logo.png?2)  
+![Stripe](https://stripe.com/img/navigation/logo.png?2)
```

Comment:
would you mind adding the double space back here, I like the build badge under the logo :)